test/e2e/knativeservingdeployment_test.go

```diff
@@ -0,0 +1,59 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/test/logstream"
+	"knative.dev/serving-operator/test"
+	"knative.dev/serving-operator/test/resources"
+	"testing"
+)
+
+// TestKnativeServingDeploymentRecreationReady verifies whether the deployment is recreated, if it is deleted.
+func TestKnativeServingDeploymentRecreationReady(t *testing.T) {
+	t.Parallel()
```

Comment:
I wouldn't run these tests in parallel. You're destroying resources in this which might affect other tests. Let's remove the Parallel flag everywhere for now. We can always add it later if test runtime becomes an issue.

Reply:
done